src/System.Private.ServiceModel/tools/test/Bridge/Bridge/IdleTimeoutManager.cs

```diff
@@ -55,6 +58,26 @@ public IdleTimeoutManager(TimeSpan idleTimeout)
                 _waitTimeout = idleTimeout;
                 _timer = new Timer(this.TimeOutCallback);
                 this.Restart();
+
+                // Register to be notified when the Bridge configuration changes
+                ConfigController.BridgeConfigurationChanged += (object s, Tuple<BridgeConfiguration, BridgeConfiguration> tuple) =>
```

Comment:
Interestingly this bleeds the configController into the idleTimeout manager. I understand this this test code but from a design principle its concepts spilling over  and we could think of the config instance class to be like viewcontroller and have an event like  `OnIdleTimeOutChanged(Timespan)` and the handler only registers to that one event. Just a bit more decoupling rather than have the entire config passed in here. 
Again don't block on these comments.